ouroboros-consensus/docs/report/unsoundswitch/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Revision history for unsoundswitch
```

Comment:
unsoundswitch is a purely internal thing; in particular, it is not published to CHaP, so I don't think it needs a changelog.